tests/functional/api/test_merge_requests.py

```diff
@@ -117,6 +117,11 @@ def test_merge_request_rebase(project):
     assert mr.rebase()
 
 
+def test_merge_request_reset_approvals(project):
+    mr = project.mergerequests.list()[0]
+    assert mr.reset_approvals()
```

Comment:
I think even just something small like this should get rid of the 401, @zampierilucas:

```suggestion
def test_merge_request_reset_approvals(gitlab_url, project):
    bot = project.access_tokens.create({"name": "bot", "scopes": ["api"]})
    bot_gitlab = gitlab.Gitlab(gitlab_url, private_token=bot.token)
    bot_project = bot_gitlab.projects.get(project.id, lazy=True)

    mr = bot_project.mergerequests.list()[0]
    assert mr.reset_approvals()
```

Reply:
Yup, that did it, I have been trying it this afternoon, but couldn't quite figure out the gitlab.Gitlab(), turns out I was missing the gitlab_url, thanks @nejch, I just pushed an update with your suggestion :D